Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.12.0-slim-bullseye
+LABEL maintainer="olgastadnik2507@gmail.com"
+ENV PYTHONUNBUFFERED=1
+WORKDIR /app
+COPY requirements.txt requirements.txt
```

Comment:
Consider specifying the destination path in the `COPY requirements.txt requirements.txt` command for clarity, even though it defaults to the current working directory (`/app`). This can help avoid confusion if the `WORKDIR` changes in the future.